doc/src/arch/reference.rst

```diff
@@ -1092,6 +1098,8 @@ They describe how a complex block interfaces with the inter-block world.
     Physical equivalence for a pin is specified by listing a pin more than once for different locations.
     For example, a LUT whose output can exit from the top and bottom of a block will have its output pin specified twice: once for the top and once for the bottom.
 
+    If the ``<pinlocations>`` tag is missing, a ``spread`` pattern is assumed.
```

Comment:
Same as with `<fc>`, Its probably best to make this more prominent (perhaps as a `.. note::`, like in `<switchblock_locations>`).